snorkel/analysis/error_analysis.py

```diff
@@ -6,33 +6,32 @@
 from .utils import to_int_label_array
 
 
-def error_buckets(
-    golds: np.ndarray, preds: np.ndarray
-) -> Dict[Tuple[int, int], List[int]]:
-    """Return data point indices bucketed by gold label/pred label combination.
+def label_buckets(*y: np.ndarray) -> Dict[Tuple[int, int], np.ndarray]:
+    """Return data point indices bucketed by label combinations.
 
-    Returned buckets[i,j] is a list of items with predicted label i and true label j.
-    For a binary problem with (1=positive, 0=negative):
-        buckets[1,1] = true positives
-        buckets[1,0] = false positives
-        buckets[0,1] = false negatives
-        buckets[0,0] = true negatives
+    A common use case is calling ``label_buckets(Y_gold, Y_pred)`` where
```

Comment:
`buckets = label_buckets...` so that when we refer to buckets[(i,j)] it's clear what that is.